code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -263,7 +263,7 @@ h1.alert, h2.alert		{color: #99aab5;}
 
 em						{font-style: normal;	font-weight: bold;}
 
-.ooc					{					font-weight: bold;}
+.ooc					{color: #3366ff;	font-weight: bold;}
```

Comment:
Oranges was using `#cca300` as an ooc color messing around and it looked pretty good. Distinct from every other color, high contrast. You should experiment with it yourself.

Reply:
Pretty sure I remember it being mentioned as looking good on darkmode but bad on lightmode.